test/unit/swaggerSupport/schemaUtilsSwagger.test.js

```diff
@@ -0,0 +1,42 @@
+const { expect } = require('chai'),
+  concreteUtils = require('../../../lib/swaggerUtils/schemaUtilsSwagger'),
+  fs = require('fs'),
+  validSwaggerFolder = './test/data/valid_swagger',
+  invalidSwaggerFolder = './test/data/invalid_swagger';
+
+describe('parseSpec method', function () {
+  it('should return true and a parsed specification', function () {
+    let fileContent = fs.readFileSync(validSwaggerFolder + '/json/sampleswagger.json', 'utf8');
+    const parsedSpec = concreteUtils.parseSpec(fileContent, {});
+    expect(parsedSpec.result).to.be.true;
```

Comment:
new line here. Can we make sure we add line after each var/let/const definition?